### PR TITLE
solution

### PR DESCRIPTION
**Pull request by tiserett**
[DEMO LINK](https://tiserett.github.io/react_person/)

### CODE REVIEW

File: src/components/Person/Person.jsx [hunk 1]
```diff
@@ -1 +1,20 @@
...
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && <p className="Person__age">{`I am ${age}`}</p>}
+    <p className="Person__partner">
+      { isMarried
+        ? `My ${sex === 'm' ? 'wife\'s' : 'husband\'s'} name is ${partnerName}`
+        : 'I am not married'
+      }
```
Comment (aleksandra-prognimak) [inline src/components/Person/Person.jsx:17]:
```suggestion
      { 
        isMarried
          ? `My ${sex === 'm' ? 'wife\'s' : 'husband\'s'} name is ${partnerName}`
          : 'I am not married'
      }
```

**Review by aleksandra-prognimak — APPROVED** [review]
🔥 (but small fix)

File: src/components/Person/Person.jsx (hunk 1 above)
Comment (tiserett) [reply src/components/Person/Person.jsx:17]: according to all other checklists, it's ok, it also looks easier for me

File: src/App.jsx [hunk 2]
```diff
@@ -25,21 +26,8 @@ export const alex = {
...
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+    <Person {...misha} />
+    <Person {...olya} />
+    <Person {...alex} />
```
Comment (AlinaMatuschak) [inline src/App.jsx:31]: `pass the whole person as a prop <Person person={misha} /> (not individual fields);`

**Review by AlinaMatuschak — APPROVED** [review]

**Pull request closed without merging**